package.json

```diff
@@ -0,0 +1,25 @@
+{
```

Comment:
So for future reference, you need to have your package.json file in the same directory as your server.js file.